### ISSUE TITLE
mainSections = []  does not work when building on github

### ISSUE THREAD

**Issue by brinkabeler** [lxndrblz/anatole#182]
**Describe the bug**
When building my site with the marketplace action linked below, I have to set mainSections = ["docs"] instead of mainSections = [] in the config in order to have content on the main page.

**To Reproduce**
1. Set up page to have content on the start page with mainSections = [] as stated on the anatole github page
2. Build page with github action
3. Homepage on github pages defaults to /posts/ subdirectory

**Expected behavior**
I expect to arrive at the contect I prepared for the homepage as I do when I build it locally.

**Additional context**
This might be connected to changing publishDir to "docs/" in order to build on github pages.
Config here: https://github.com/brinkabeler/brinkabeler.github.io/blob/main/config.toml

**Comment by lxndrblz** [comment]
@brinkabeler Thanks for bringing this up!. I am sorry, but I could not follow why you changed the `publishDir`. The `publishDir` only changes the directory -to which the output (static site) is generated- from `public` to whatever you specify. This parameter is not theme specific and does not control the content on the front page.

Did I understand you correctly, that you simply wanted to show the content in `docs` folder on your frontpage? However, from your GitHub repo it looks, like you currently do not have a docs folder with content.

Can you please shed some light on this?

**Comment by brinkabeler** [comment]
Sorry about the confusion, those thing are probably not related.

I want to show the content in my content/_index.md on the frontpage. This works locally by setting mainSections = [] in the config.

When I build my page on github I have to set mainSections = ["docs"] to get the same results. Otherwise github pages tries to load /posts/ instead of the main page. I did this because I saw this in an example on the theme page readme, not knowing what it would to but trying everthing to get the page working... So I don't know what it was supposed to do, just that it fixed my problem.

Setting publishDir to "docs/" is a requirement of github pages. This might not be related, but I thought it might be helpful addional context. It seems not to be ;)
